systems/plants/RigidBody.cpp

```diff
@@ -8,6 +8,7 @@ const int defaultRobotNum[1] = {0};
 const set<int> RigidBody::defaultRobotNumSet(defaultRobotNum,defaultRobotNum+1);
 
 RigidBody::RigidBody(void) :
+    parent(nullptr),
```

Comment:
nullptr is a c++11 exclusive keyword.  I noticed in the makefiles there is support for compilation under C99 where this might break.